pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -99,6 +100,10 @@ def on_evaluation_end(self, *args, **kwargs):
         else:
             self.trainer.call_hook('on_validation_end', *args, **kwargs)
 
+        if self.trainer.state != TrainerState.FITTING:
+            # summarize profile results
+            self.trainer.profiler.describe()
```

Comment:
Let's add a there for test and validate. Don't we call describe twice for test ?

Reply:
done